src/app/components/Application/Application.jsx

```diff
@@ -33,7 +33,10 @@ class Application extends React.Component {
   }
 
   getChildContext() {
-    return { media: this.state.media };
+    return {
+      media: this.state.media,
+      includeDrbb: appConfig.includeDrbb,
```

Comment:
Maybe my ignorance - does this need to be in `contextTypes` ? I feel like it would be better to access the environment variable directly, perhaps by importing it into `appConfig.js`.  Then you can access the variable as the static const that it is.